app/sim_test.go

```diff
@@ -312,12 +311,12 @@ func TestAppStateDeterminism(t *testing.T) {
 	config.AllInvariants = false
 	config.ChainID = helpers.SimAppChainID
 
-	numSeeds := 3
-	numTimesToRunPerSeed := 5
+	numSeeds := config.NumBlocks / 10
+	numTimesToRunPerSeed := 2
 	appHashList := make([]json.RawMessage, numTimesToRunPerSeed)
 
 	for i := 0; i < numSeeds; i++ {
-		config.Seed = rand.Int63()
+		config.Seed = config.Seed + int64(i)
```

Comment:
Why don't we use `rand.Int63()`?

Reply:
In the `test-sim-nondeterminism` used in ci/cd workflow, if the result of each ci/cd is not the same by random seed may cause confusion, so the rand was removed and the seed was fixed to an [explicit value](https://github.com/crescent-network/crescent/pull/175/files/352d813a376975315b87b418781cca230f1bc38b#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R197)